packages/calcite-components/src/tests/commonTests.ts

```diff
@@ -965,125 +959,178 @@ export async function disabled(
     }
   }
 
-  expect(component.getAttribute("aria-disabled")).toBeNull();
+  // only testing events from https://github.com/web-platform-tests/wpt/blob/master/html/semantics/disabled-elements/event-propagate-disabled.tentative.html#L66
+  const eventsExpectedToBubble = ["mousemove", "pointermove", "pointerdown", "pointerup"];
+  const eventsExpectedToNotBubble = ["mousedown", "mouseup", "click"];
+  const allExpectedEvents = [...eventsExpectedToBubble, ...eventsExpectedToNotBubble];
 
-  if (options.focusTarget === "none") {
-    await page.click(tag);
-    await expectToBeFocused("body");
+  const createEventSpiesForExpectedEvents = async (component: E2EElement): Promise<EventSpy[]> => {
+    const eventSpies: EventSpy[] = [];
 
-    assertOnMouseAndPointerEvents(eventSpies, (spy) => expect(spy).toHaveReceivedEventTimes(1));
+    for (const event of allExpectedEvents) {
+      eventSpies.push(await component.spyOnEvent(event));
+    }
+    return eventSpies;
```

Comment:
Nitpick: add newline above the return to preserve spacing between function parts.